spec/policies/hospital_policy_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe HospitalPolicy do
```

Comment:
What is missing in this spec?

Reply:
I think, only admin users should be able to manage hospitals, some similar behaviour should be applied to other specs